### PR TITLE
Issue: #4097 Dropped CPUID Support

### PR DESCRIPTION
**Pull request by arunsethupat**
Dropped CPUID Support

### CODE REVIEW

File: cmake/FindCpuid.cmake [hunk 1]
```diff
@@ -10,24 +10,24 @@
...
-
-set(CPUID_INCLUDE_DIRS ${CPUID_INCLUDE_DIR})
-set(CPUID_LIBRARIES ${CPUID_LIBRARY})
-
-# handle the QUIETLY and REQUIRED arguments and set CPUID_FOUND to TRUE
-# if all listed variables are TRUE, hide their existence from configuration view
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(Cpuid DEFAULT_MSG CPUID_LIBRARY CPUID_INCLUDE_DIR)
-mark_as_advanced (CPUID_INCLUDE_DIR CPUID_LIBRARY)
+#find_path(
```
Comment (chfast) [inline cmake/FindCpuid.cmake]: Remove this file.

File: cmake/UseCpuid.cmake [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-	eth_show_dependency(CPUID Cpuid)
-	if (CPUID_FOUND)
-		target_include_directories(${TARGET} SYSTEM PUBLIC ${CPUID_INCLUDE_DIRS})
-		target_link_libraries(${TARGET} ${CPUID_LIBRARIES})
-		target_compile_definitions(${TARGET} PUBLIC ETH_CPUID)
-	elseif (NOT ${REQUIRED} STREQUAL "OPTIONAL")
-		message(FATAL_ERROR "CPUID library not found")
-	endif()
-endfunction()
+#function(eth_apply TARGET REQUIRED)
```
Comment (chfast) [inline cmake/UseCpuid.cmake]: Remove this file.

**Comment by arunsethupat** [comment]
Removed the files and looked for the `find_package(Cpuid)`. Kindly check.

**Comment by chfast** [comment]
Nice.

Your commits does not have author email set so they are not linked to your GitHub accounts. It does not matter to me, but maybe you want to be properly listed as a contributor.

**Comment by arunsethupat** [comment]
Oh, Did not know this. Could you help me set author email please. :)

**Comment by chfast** [comment]
First set git config for user:
```
$ git config --global user.name "John Doe"
$ git config --global user.email johndoe@example.com
```
It does not have to be the GitHub login email, you can add multiple emails to GitHub account.

Then you have to amend the 2 commits. Try this: https://stackoverflow.com/questions/3042437/change-commit-author-at-one-specific-commit.

**Comment by arunsethupat** [comment]
Thanks @chfast. Followed the steps and amended the commits. Could you please confirm if you can see it now.

**Comment by chfast** [comment]
You have to push them now. `git push -f`.

**Comment by arunsethupat** [comment]
git push -f to the forked repository? or the main one? Pushed to my forked repository now, but couldn't push to this ethereum/cpp. It says Im unauthorised.

**Comment by chfast** [comment]
Ok, but you also pushed one of my commits. See https://github.com/ethereum/cpp-ethereum/pull/4298/commits.

**Comment by arunsethupat** [comment]
Made the change. Steep learning curve :)

**Comment by chfast** [comment]
Yes, awesome work! Especially, if this is your first contact with git.

**Comment by arunsethupat** [comment]
Thank you. :) But why did the checks fail? What do I do now?

**Comment by chfast** [comment]
All perfect, just single code coverage check failed. But this was not an issue when code was removed.

**Pull request merged**